src/book/03-guides/01-main-codebase/01-getting-started.mdx

```diff
@@ -26,15 +30,19 @@ There is a guide to using Git in NUbots [here](/guides/general/git).
 
 Docker is a container system that allows you to install development tools and run builds in an isolated environment that won't affect the rest of your system. We use it to set up the NUbots build system and run builds and binaries.
 
-#### Ubuntu
+You will need to perform the steps correctly to set up Docker for your operating system to complete the example.
+
+<details>
+
+<summary>Ubuntu</summary>
```

Comment:
Since we specify `focal`, should we change this heading to `Ubuntu 20.04`?

Also, this section has headings for Mint, so should that be included in the heading too? We are assuming that these people won't know to even click on this tab, because they won't know the relationship between mint and ubuntu.

Reply:
I've added info on what to change this too if you aren't using focal fossa. Also changed the drop-down heading to Ubuntu/Linux Mint.